docs-api/documentation.md

```diff
@@ -0,0 +1,370 @@
+coremltools API v4.1 Documentation
+==================================
+
+This document describes the following:
+
+* [Viewing documentation](#viewing-documentation)
+* [Updating the documentation](#updating-the-documentation)
+* [Generating a new API reference](#generating-a-new-api-reference)
+
+
+## Viewing documentation {#viewing-documentation}
+
+For coremltools documentation, see the following:
+
+* Core ML Tools [README](README.md) file for this repository
+* [Guides and examples](https://coremltools.readme.io/) with installation and troubleshooting
+* [API Reference](https://apple.github.io/coremltools/index.html)
+* [Building from Source](https://github.com/apple/coremltools/blob/master/BUILDING.md)
+* [Release Notes](https://github.com/apple/coremltools/releases/) for the current release and previous releases
+* [Previous coremltools API (v3.4)](https://apple.github.io/coremltools/docs-api-version3/index.html)
+
+
+## Updating the documentation {#updating-the-documentation}
+
+You can update the following:
+
+### Updating the guides and examples
+
+To make editing changes to [Guides and examples](https://coremltools.readme.io/), see [the Documentation section of Contributing](https://coremltools.readme.io/docs/how-to-contribute#documentation).
+
+### Updating the API reference
+
+To create a new version or update the HTML files for the API reference, follow these general steps:
+
+1. Fork and clone the repo and edit the content as usual.
+2. Generate the HTML files by running this script from the root folder: `zsh -i scripts/build_docs.sh`. The HTML files appear in the `docs/_build/html` folder.
+3. Copy the `html` folder to a temporary location, and use it to preview your changes. Repeat 1-3 until finished.
+4. Push the changes to your fork, and create a pull request (PR) to merge them with the `main` branch of Apple's repo.
+5. Create and checkout the `gh-pages` branch from upstream.
```

Comment:
I would just say checkout. You shouldn't be creating it. It should already exist.